imsim/stamp.py

```diff
@@ -68,7 +68,8 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
             # For really faint things, don't try too hard.  Just use 32x32.
             image_size = 32
 
-        elif isinstance(gal, galsim.DeltaFunction):
+        elif (not isinstance(gal, galsim.ChromaticSum)
```

Comment:
I assume this part is just to make sure `gal._original` doesn't give an AttributeError.  So probably better to write this as:
```
elif (hasattr(gal, 'original') and isinstance(gal.original, galsim.DeltaFunction)):
```
Also I removed the leading underscore.  The intended API is for users to use `gal.original`, which is a read-only property returning what is currently implemented as a private attribute, `gal._original`.

Reply:
Thanks Mike!  I've implemented this change.